src/backend/TrafficCourts/Citizen.Service/Models/ViolationTicket.cs

```diff
@@ -0,0 +1,43 @@
+/// <summary>
+/// A model representation of the extracted OCR results.
+/// </summary>
+public class ViolationTicket
+{
+    public ViolationTicket() { }
+
+    /// <summary>
+    /// Confidence of correctly extracting the document.
+    /// </summary>
+    public float confidence { get; set; }

```

Comment:
PascalCase... for data being returned, the serialize can make the properties camelCased via configuration or by using the `[JsonPropertyName("confidence")]` attribute.